_opensearch/rest-api/index-apis/clear-index-cache.md

```diff
@@ -8,34 +8,38 @@ nav_order: 320
 
 ## Clear index or data stream cache
 
-Clears the caches of one or more indexes. For data streams, the API clears the caches of the stream’s backing indexes.
+The cache clear API operation clears the caches of one or more indexes. For data streams, the API clears the caches of the stream’s backing indexes.
```

Comment:
"clear cache API"? That's how it's used below and I think it is more understandable in that order.

Reply:
thanks!